frontend/ui/widget/buttondialog.lua

```diff
@@ -54,6 +54,7 @@ local ScrollableContainer = require("ui/widget/container/scrollablecontainer")
 local Size = require("ui/size")
 local TextBoxWidget = require("ui/widget/textboxwidget")
 local UIManager = require("ui/uimanager")
+local util = require("util")
```

Comment:
All the way at the bottom please, see https://github.com/koreader/koreader-base/wiki/Coding-style#order-of-requires